primitives/npos-elections/compact/tests/compact.rs

```diff
@@ -0,0 +1,168 @@
+use std::{
+	collections::{HashSet, HashMap},
+	convert::TryInto,
+	hash::Hash,
+};
+use sp_npos_elections::{CompactSolution, IndexAssignment};
+use rand::{self, Rng, SeedableRng, seq::SliceRandom};
+
+const CANDIDATES: usize = 1000;
+const VOTERS: usize = 2500;
+
+sp_npos_elections_compact::generate_solution_type!(
+	#[compact]
+	pub struct Compact::<VoterIndex = u32, TargetIndex = u16, Accuracy = Accuracy>(16)
+);
+
+pub type AccountId = u64;
```

Comment:
unlike pallet stuff, in this case I have to ask why these are pub

Reply:
Outdated now, but the intent was to be able to share around the helper functions for the test code.